src/Job/StaticCacheFullBuildJob.php

```diff
@@ -0,0 +1,100 @@
+<?php
+
+namespace SilverStripe\StaticPublishQueue\Job;
+
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\StaticPublishQueue\Contract\StaticallyPublishable;
+use SilverStripe\StaticPublishQueue\Extension\Publishable\PublishableSiteTree;
+use SilverStripe\StaticPublishQueue\Job;
+use SilverStripe\StaticPublishQueue\Publisher;
+use SilverStripe\Versioned\Versioned;
+
+/**
+ * Adds all pages to the queue for caching. Best implemented on a cron via StaticCacheFullBuildTask.
+ */
+class StaticCacheFullBuildJob extends Job
+{
+    /**
+     * @return string
+     */
+    public function getTitle()
+    {
+        return 'Generate a static pages for all URLs';
```

Comment:
This title is pretty similar to GenerateStaticCacheJob - maybe could use clarification